### DISCUSSION TITLE
New demo release 11.11.1

### DISCUSSION THREAD

**Discussion by palexdev** [palexdev/MaterialFX#46] [Archived]
New minor update, fixed many bugs, improved some controls behavior

<hr /><em>This discussion was created from the release <a href='https://github.com/palexdev/MaterialFX/releases/tag/v11.11.1'>New demo release 11.11.1</a>.</em>

**Comment by ghost** [comment]
Hello i have a question, its only possible use the library by importing it with gradle or maven?
There is no any .jar file to import it manually?

**Reply by palexdev** [reply]
Yes it is supposed to be used with a build system
To use the .jar you must clone and build the project yourself, it will produce three jar for the library, the source code and the javadoc

**Reply by ghost** [reply]
Okay, im pretty new with JavaFX and im having some troubles with deploying my app with maven/gradle so i started it without a build system.

**Reply by palexdev** [reply]
> Okay, im pretty new with JavaFX and im having some troubles with deploying my app with maven/gradle so i started it without a build system.

Yeah I've been there too, however I suggest you to put the initial effort on learning Gradle as it is much easier to use JavaFX and manage all the needed dependencies with a build system

**Reply by ghost** [reply]
Thanks for the reply, i will try to start my project with gradle.

**Reply by ghost** [reply]
@palexdev Hey, i did maked the project work with gradle. Thanks for ur comment.
I have another question, How i can implement the stepper that u have in the readme page of the project?

**Reply by palexdev** [reply]
> @palexdev Hey, i did maked the project work with gradle. Thanks for ur comment.
> I have another question, How i can implement the stepper that u have in the readme page of the project?

The logic is implemented here: [Controller](https://github.com/palexdev/MaterialFX/blob/main/demo/src/main/java/io/github/palexdev/materialfx/demo/controllers/StepperDemoController.java)
The view is made with SceneBuilder
